classes/controller/FrontController.php

```diff
@@ -1239,17 +1229,10 @@ protected function getOverrideThemeDir()
      */
     public function getLayout()
     {
-        $layout = 'layout/layout-full-width.tpl';
-
-        if ($this->display_column_left && $this->display_column_right) {
-            $layout = 'layout/layout-both-columns.tpl';
-        } elseif ($this->display_column_left) {
-            $layout = 'layout/layout-left-column.tpl';
-        } elseif ($this->display_column_right) {
-            $layout = 'layout/layout-right-column.tpl';
-        }
-
         $entity = $this->php_self;
+
+        $layout = $this->context->shop->theme->getLayoutForPage($entity);
```

Comment:
I renamed it inside getLayoutForPage() but I like $entity here because of the [next line](https://github.com/julienbourdeau/PrestaShop/blob/starter-theme/feat/theme-management/classes/controller/FrontController.php#L1236)
```php
$id_item = (int)Tools::getValue('id_'.$entity);
```